src/schema/responses.ts

```diff
@@ -168,6 +171,8 @@ function parseResponseData({cmd, data}: zod.infer<typeof ResponseFrame>) {
       return UserSettingsGetLocaleResponse.parse(data);
     case Commands.OPEN_EXTERNAL_LINK:
       return OpenExternalLinkResponse.parse(data);
+    case Commands.GET_RELATIONSHIPS:
+      return GetRelationshipsResponse.parse(data);
```

Comment:
dupe since L191 includes this generated response

Reply:
whoops, good catch